android/app/src/main/python/electroncash_gui/android/strings.py

```diff
@@ -41,3 +41,5 @@ def _(s):
 _("Use a master key")
 _("Wallet seed")
 _("You don't have any contacts.")
+_("Rename wallet")
+_("Wallet names cannot contain the '/' character. Please enter a different wallet name to proceed.")
```

Comment:
Please keep this file in alphabetical order.

Reply:
Might also be worth adding a comment to remind people to do that in future .

Reply:
Ah, not sure how I missed this!  There were a handful of other entries out of order I've also sorted, and added a comment as you suggest.